rust/operator-binary/src/main.rs

```diff
@@ -50,8 +53,8 @@ async fn main() -> anyhow::Result<()> {
     let opts = Opts::parse();
     match opts.cmd {
         Command::Crd => {
-            TrinoCluster::print_yaml_schema(built_info::PKG_VERSION)?;
-            TrinoCatalog::print_yaml_schema(built_info::PKG_VERSION)?;
+            v1alpha1::TrinoCluster::print_yaml_schema(built_info::PKG_VERSION)?;
+            crd::catalog::v1alpha1::TrinoCatalog::print_yaml_schema(built_info::PKG_VERSION)?;
```

Comment:
This should use the merged_crd method, otherwise a future version won't be emitted.

See: https://github.com/stackabletech/opa-operator/blob/e17686ac790faaaf69ed041b6f19109e31f44fd7/rust/operator-binary/src/main.rs#L60-L63

Reply:
[done](https://github.com/stackabletech/trino-operator/pull/711/commits/cdc58bc54a61a0faba1375af2b0d90173c5a036d)